staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -624,16 +624,14 @@ See https://kubernetes.io/docs/reference/using-api/server-side-apply/#conflicts`
 		if err != nil {
 			return err
 		}
-		patchBytes, patchedObject, err := patcher.Patch(info.Object, modified, info.Source, info.Namespace, info.Name, o.ErrOut)
+		changed, err := patcher.Patch(info, modified, o.ErrOut)
 		if err != nil {
-			return cmdutil.AddSourceToErr(fmt.Sprintf("applying patch:\n%s\nto:\n%v\nfor:", patchBytes, info), info.Source, err)
```

Comment:
You've removed this wrapping but now there's nowhere that the `patchBytes` will appear in the error. I do think it was helpful for the user to see the patch being applied in the message as well as getting overall context from the operation. Since this is a code cleanup I think it'd be a shame to lose the functionality